### PR TITLE
Feature/bing 61 design search dropdown

### PR DESCRIPTION
**Pull request by palianskas**
Added dropdown to appbar-search to display results (mocked at the moment). On dropdown item click a _title-dialog_  is opened with the data from the dropdown item's title

### CODE REVIEW

**Review by MatasRut — COMMENTED** [review]
CRLF line ending errors appear on every file changed. Reformat files and make sure git is not converting line endings

**Review by MatasRut — CHANGES_REQUESTED** [review]
Search dropdown items should have a hover effect and the mouse cursor should change to the clicking hand. Right now they do not look clickable.

**Comment by palianskas** [comment]
> CRLF line ending errors appear on every file changed. Reformat files and make sure git is not converting line endings

Seems like a false flag. Marking as resolved

> Search dropdown items should have a hover effect and the mouse cursor should change to the clicking hand. Right now they do not look clickable.

Moved dropdown's `handleClose` to a different callback, fixed in latest commit

File: app/bingey-ui/src/components/Watchlist/TitleDialog/titleDialogStyle.scss [hunk 1]
```diff
@@ -1,22 +1,28 @@
-.title{
-    margin-right: auto !important;
+.title {
```
Comment (palianskas) [inline app/bingey-ui/src/components/Watchlist/TitleDialog/titleDialogStyle.scss]: Add overflow ellipsis to long title names

**Review by MatasRut — APPROVED** [review]

**Pull request merged**